docs/_articles/en/user-guide/perf-enhancements.md

```diff
@@ -3,37 +3,32 @@ title: Performance Enhancements
 lang: en
 ---
 
-The Apex, Deploy and Retrieve commands are in the process of moving to a new architecture. These commands have been rearchitected into independent Typescript libraries and CLI plugins as part of our larger effort to break up the salesforce-alm plugin.  [You can read more about that in this blog post](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021.html). There’s many great benefits to this effort and one of the best is better performance. For single file deploys and retrieves, you'll likely notice a significant performance gain when using the latest architecture.
+The Apex, Deploy and Retrieve commands are in the process of moving to a new architecture. These commands have been rearchitected into independent Typescript libraries and CLI plugins as part of our larger effort to break up the salesforce-alm plugin. [You can read more about that in this blog post](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021.html). There’s many great benefits to this effort and one of the best is better performance. For single file deploys and retrieves, you'll likely notice a significant performance gain when using the latest architecture.
 
 ## Execution Process and Status
 
-This architecture change is done through a phased approach.  For both the Apex library and Deploy/Retrieve library, we are rolling out the changes through these phases:
+This architecture change is done through a phased approach. For both the Apex library and Deploy/Retrieve library, we are rolling out the changes through these phases:
 
 1. Library created and completed.
-2. Available to try it in VS Code - We provide a VS Code setting so you can opt in and try it out. 
-3. On by default in VS Code - The VS code setting default is defaulted to True to enable the library. If you previously opted in and then opted out, you'll have to manually opt back in to use the feature.  We'll change the default after the commands have been available and used for months.
-4. CLI updated to use the library.  The VS Code setting is then retired.  Both the CLI and VS Code extensions are using the library.
+2. Available to try it in VS Code - We provide a VS Code setting so you can opt in and try it out.
```

Comment:
We should also make the same changes to the article in the Japanese doc path since it's currently in English.

Reply:
Good idea, updated